popup.js

```diff
@@ -50,6 +62,17 @@ document.addEventListener('DOMContentLoaded', async () => {
       chrome.tabs.sendMessage(tab.id, { type: MSG_REVERT_C2PA_INDICATOR });
     }
   });
+
+  computeDataURLcheckbox.addEventListener('change', async () => {
+    // Save the state to storage
+    chrome.storage.local.set({ computeDataURL: computeDataURLcheckbox.checked });
+
+    if (computeDataURLcheckbox.checked) {
+      chrome.tabs.sendMessage(tab.id, { type: MSG_COMPUTE_DATA_URL });
```

Comment:
in this case, we also need to trigger the readC2pa function for gated images. I might be wrong, but it doesn't seem to be handled